### DISCUSSION TITLE
BPT E272 compatibility

### DISCUSSION THREAD

**Discussion by ppkstat** [PricelessToolkit/ESPBell-MAX#7] [General]
I wanted to make sure if this is compatible with my intercom before bying. I am attaching the diagram from the manual and a picture of my acual own. 
I am not very familiar with intercoms but if I am geeting it right it should be DB1, R1A -> 5, DB2-> 7 (or perhaps 6 I am not sure), R1B->8. Is everything correct? Will it work along with my intercom in this case?

Also the aliexpress links on the main page are dead. I just wanted to know what mounting hardware should I get if I prin the case on my own. Thank you! 
![diag](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/738fab1c-ba83-4ef1-b743-134a7d9569da)
![1715707409478](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/fd9cfdc1-84af-47d3-9c2a-8cecd57261a0)

**Comment by PricelessToolkit** [comment]
Hi, Yes it is a 4+1 analog system and you got it right, 5 Is common, so 
"DB1, R1A -> 5"
"DB2 -> 7 not 6!"
"R1B -> 8"

**Comment by ppkstat** [comment]
Thank you, I just placed the order! Could you please tell me what the length for the m2 inserts? The Alliexpress links are dead.

**Reply by PricelessToolkit** [reply]
Link is working, length is 3.5mm

**Comment by ppkstat** [comment]
I received and flashed/installed the device without issues. However when I came to install it I realised that there is no '7' position on the screw terminal as you can see from the picture. On the schematic the '7' terminal also appears to be separate, am I missing something?

**Reply by PricelessToolkit** [reply]
Hi, please do a closeup photo of the entire interior, and yes 7 Terminal is a separate one. 

![Θυροτηλέφωνο-bpt-e272](https://github.com/PricelessToolkit/ESPBell-MAX/assets/81021972/f41b2ac1-b84a-4146-9829-10188c5346aa)

**Reply by PricelessToolkit** [reply]
I think it's a white wire.

![330534718-fd9cfdc1-84af-47d3-9c2a-8cecd57261a0](https://github.com/PricelessToolkit/ESPBell-MAX/assets/81021972/95d18b85-a154-40ed-abb7-dc1f32c1f014)

**Comment by ppkstat** [comment]
Yes that must be it. It's marked as '2' in my case.
I did some testing earlier by manually triggering the device and while the device was triggered without issues the notification on my phone was arriving app. 10 seconds later. By the time I could see the notification the device has gone to sleep so I could not test the relays. But hopefully I 'll be doing some real world testing tomorrow. 
![1717109100032](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/c47f8a57-d849-4521-b8f5-6978ecca555d)
![1717109278093](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/d22e840d-e3ee-4b0b-a3d3-b7910ac73609)

**Reply by PricelessToolkit** [reply]
Normally, notifications arrive in about 0.2-0.250 milliseconds. If they are delayed, there might be something wrong with the configuration. Check the MAC address and ensure the IP address is outside of the DHCP range.

**Reply by ppkstat** [reply]
Right ok, did some real world testing today. What happens is that the device is triggered and wakes up by the doorbell but I am not receiving the notification. As soon as I power cycle the device I receive the notification immediately. The ip is outside of dhcp range and the mac address seems to be right. In fact I tried 2 addresses (for two different access points) and I could see the device connected in each.

**Reply by PricelessToolkit** [reply]
1. All LED is off when you trigger the board?
2. Can you check after triggering how much time passes until the status is changed in "Devices & Services / MQTT / ESPBell-MAX"  and one more thing, the default status is "Closed"?
3. Have you changed any time-based settings?

it is odd because the first thing ESPBell does is publish the "open" status, and before shutting off publishes the "close" status.

![bell](https://github.com/PricelessToolkit/ESPBell-MAX/assets/81021972/12a2c45c-af98-42a3-af25-7ef5703150d3)

**Comment by ppkstat** [comment]
1. Yes they are off. Power led goes on when triggered (and then returns to
off again).

2. Approximately 10 seconds. It's pretty consistent each time. It goes from
closed to open and I get the notification.

3. All timing settings in config.h are the default ones.

Thank you!

**Reply by PricelessToolkit** [reply]
Okay, 

1. Can you launch a ping in the terminal (cmd) using the command "ping 192.168.x.x -t" ? Replace "192.168.x.x" with the ESPBell IP address. Then trigger the board to see how much time it takes to connect to WiFi.

2. Please check the topic "homeassistant/binary_sensor/espbell-max/state" using MQTT Explorer (https://mqtt-explorer.com/). Verify if the default status is "OFF" or not.

The expected status changes are:

OFF - Default
ON - Triggered
OFF - Before sleeping, publishes "OFF" (approximately 14 seconds after being triggered).

3. Can you publish your config? Do not mask IP addresses as they are local IPs and don't need to be masked. Leave 50% of the MAC address visible as well.

**Reply by ppkstat** [reply]
1. It takes the same amount of time app 10 seconds. At least we found its a wi-fi issue. No idea why though. There are sometimes in which the device seems to not be able to connect at all and the led stays on. I have to power cycle at that point. 
2. Yes this is sort of the behaviour I am getting. OFF to ON is slow (as explained in point to 1) and ON to OFF is instantaneous. It seems that off seems to be published twice, see attached screenshot. I am not very familiar with MQTT Explorer.
3. Below you can see the config. Do note that the IP address is outside of the DHCP range.

![mqqt](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/0c9d6675-9990-4fdc-b07e-88a0e39ba2eb)
![trou](https://github.com/PricelessToolkit/ESPBell-MAX/assets/138029001/efac5ce7-8e76-44ec-a9c2-bfcf1021fbd8)

**Reply by PricelessToolkit** [reply]
If it takes 10 seconds to connect to the Wi-Fi, it suggests that the device is scanning for available networks before connecting, which is unusually slow. Typically, scanning and connecting should take only 2-5 seconds. Under normal circumstances, the device will not scan if the MAC address, channel, and SSID, are correct.

The problem could be one of the following:

1. MAC Address: You mentioned it is correct.
2. Channel: The channel configuration might be causing delays.
3. Multiple Access Points with the Same Name: This can confuse the device.
4. Mesh Network: In a mesh network, the client device automatically connects to the strongest signal, which can cause delays.
5. Band Steering Enabled: If band steering is enabled, the access point may be trying to switch the client connection to 5.8 GHz, leading to a longer connection time.

If none of these solutions work for you, you can return the device. I don't want you to keep it and use it as a paperweight.

**Comment by ppkstat** [comment]
Unfortunately I was never able to make it work. Concerning my network and your points:

1. I triple checked the MAC addresses and tried two different ones for my two APs. Each time I was changing the address the device was connecting (eventually) to the corresponding AP. 
2. The channel for 2.4GHz is 11 and it's not changing because I also have a zigbee network. 
3. They don't have the same name. They do share the same SSIDs, as they should. 
4. There is no Meshing all APs are hardwired. 
5. Band Steering is disabled. 

I wanted a solution so I got inspired by your design and made a device myself with a D1 mini and an SSR relay, very similar to ESP Bell lite (but much larger of course). I even used your automations, I hope you don't mind! The device works flawlessly. 

I would return the BellMax but I don't think I can because during testing I made a booboo and shorted something (because the cable harness has naked ends). The trace between the B+ (if I remember correctly) and the optocoupler was fried. I repaired the pcb with some wire wrap and the device is working like before, but still..

**Reply by PricelessToolkit** [reply]
I am glad you found the solution :) please contact me taught shop, and include a link to this conversation.
